### PR TITLE
chore: remove old feat flags

### LINKED ISSUE #2995: Remove old feature flags
### Overview

We want to remove the feature flags for

* `OnRamperFiatRamp`
* `MtPelerinFiatRamp`
* `RainbowCharts`

### References and additional details

* An example of how to do this https://github.com/shapeshift/web/pull/2861

### Acceptance Criteria

* There are no remnants or dead code associated with these flags, including environment variables
* All of these features still work as they currently do

### Need By Date

n/a

### Screenshots/Mockups

image

### Ownership

- [X] If my bounty needs engineering or needs product I have added the respective labels on the right
- [X] As the sponsor of this bounty I will review the changes in a preview environment (ops/product) or review the PR (engineering)

### Estimated effort

2 hours

### Sponsor / Stakeholder

0xdef1cafe

### Bounty Hunters

- [Join our discord](https://discord.gg/shapeshift)
- Include an expected timeline for you to complete work in the work plan when you apply for this bounty!
- Please refer to [this link](https://shapeshiftdao.gitbook.io/getting-started/how-to-contribute/bounties#resources-for-new-contributors-and-bounty-hunters) for some basic info
- Please ***do not*** start work on this issue until you are approved in Gitcoin.

### PR DESCRIPTION
**Pull request by brymut**
## Description

Removing old feat flags for:
- `OnRamperFiatRamp`
- `MtPelerinFiatRamp`
- `RainbowCharts`

Using #2861 as guidance reference.

## Notice

- [x] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/blob/develop/CONTRIBUTING.md) guide?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [x] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [ ] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

closes #2995 

## Risk

Lower risk, dealing with technical debt removing old feat flags.

## Testing

- Check list of

### CODE REVIEW

**Review by 0xdef1cafe — CHANGES_REQUESTED** [review]
@brymut this is completely wrong - did you even read the issue?

**Comment by brymut** [comment]
> @brymut this is completely wrong - did you even read the issue?

Sorry about that @0xdef1cafe , forgot about the second A/C when I started working. Will fix asap

**Comment by brymut** [comment]
Could you please have another look @0xdef1cafe ?

File: src/config.ts [hunk 1]
```diff
@@ -64,9 +64,7 @@ const validators = {
   REACT_APP_FEATURE_COWSWAP: bool({ default: false }),
   REACT_APP_FEATURE_YAT: bool({ default: false }),
   REACT_APP_FEATURE_AXELAR: bool({ default: false }),
-  REACT_APP_FEATURE_RAINBOW_CHARTS: bool({ default: false }),
   REACT_APP_FEATURE_MTPELERIN_FIAT_RAMP: bool({ default: false }),
```
Comment (0xdef1cafe) [inline src/config.ts:67]: ?

**Review by 0xdef1cafe — CHANGES_REQUESTED** [review]
@brymut

File: src/config.ts (hunk 1 above)
Comment (brymut) [reply src/config.ts]: Sorry, having a bad day 😓

**Review by 0xApotheosis — APPROVED** [review]
`OnRamperFiatRamp`, `MtPelerinFiatRamp`, and `RainbowCharts` feature flags no longer appear in codebase, and their respective functionality still works as expected.

**Review by 0xdef1cafe — APPROVED** [review]

**Pull request merged**